public/app/features/alerting/unified/hooks/useAbilities.ts

```diff
@@ -173,6 +179,25 @@ export const useAlertingAbility = (action: AlertingAction): Ability => {
   return allAbilities[action];
 };
 
+/**
+ * This one will check for enrichment abilities
+ */
+export const useEnrichmentAbilities = (): Abilities<EnrichmentAction> => {
+  const userIsAdmin = isAdmin();
+  const hasReadPermission = ctx.hasPermission(AccessControlAction.AlertingEnrichmentsRead);
+  const hasWritePermission = ctx.hasPermission(AccessControlAction.AlertingEnrichmentsWrite);
+
+  return {
+    [EnrichmentAction.Read]: [AlwaysSupported, userIsAdmin || hasReadPermission],
+    [EnrichmentAction.Write]: [AlwaysSupported, userIsAdmin || hasWritePermission],
```

Comment:
I do wonder if the first value should be `AlwaysSupported`. 
The first value describes if the component is available in the system (like Ruler API or ALertmanager API)

Maybe the value here should be `config.featureToggles.alertEnrichment`? Also enrichments doesn't work in OSS, so `AlwaysSupported` feels a bit misleading here

It's not a blocker, just something we might want to consider

Reply:
yes, you're right.